### PR TITLE
Linking to new forms.

### PR DESCRIPTION
**Pull request by gvwilson**
1. Adding configuration variable for AMY.
2. Modifying workshop request page to link to SWC and DC request forms.
3. Modifying 'join' page to include link to instructor info update form.

@pbaaszkiewicz please review.

### CODE REVIEW

**Comment by pbanaszkiewicz** [comment]
@gvwilson is this live on dev.software-carpentry.org?

**Comment by gvwilson** [comment]
Nope - not enough disk space to build it.

On 2015-09-24 9:51 AM, Piotr Banaszkiewicz wrote:
>
> @gvwilson <https://github.com/gvwilson> is this live on 
> dev.software-carpentry.org?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/swcarpentry/site/pull/1097#issuecomment-142860628>.
>

-- 
Dr. Greg Wilson    | gvwilson@software-carpentry.org
Software Carpentry | http://software-carpentry.org

**Comment by gvwilson** [comment]
but it looks OK on my machine

On 2015-09-24 9:51 AM, Piotr Banaszkiewicz wrote:
>
> @gvwilson <https://github.com/gvwilson> is this live on 
> dev.software-carpentry.org?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/swcarpentry/site/pull/1097#issuecomment-142860628>.
>

-- 
Dr. Greg Wilson    | gvwilson@software-carpentry.org
Software Carpentry | http://software-carpentry.org

File: workshops/request.html [hunk 1]
```diff
@@ -39,189 +53,8 @@
...
-<input type="hidden" name="pageHistory" value="0">
-<input type="hidden" name="fbzx" value="-4405048771215045936">
-<div class="ss-item ss-navigate"><table id="navigation-table"><tbody><tr><td class="ss-form-entry goog-inline-block" id="navigation-buttons" dir="ltr">
-<input type="submit" name="submit" value="Submit" id="ss-submit" class="jfk-button jfk-button-action ">
-<div class="ss-password-warning ss-secondary-text">Never submit passwords through Google Forms.</div></td>
-</tr></tbody></table></div></ol></form></div>
+<p>
+  We strive to be a global project and support diversity in science.
+  If you wish to offer a workshop that would further these goals,
+  please <a href="{{site.contact}}">contact us</a> regarding a waiver for the administration fee.
```
Comment (pbanaszkiewicz) [inline workshops/request.html]: I would add link to the administrative fee page here, but that's not a critical issue in this PR.

**Comment by pbanaszkiewicz** [comment]
+1 to merge, you can add the link or not.

**Pull request merged**